mm2src/common/jsonrpc_client.rs

```diff
@@ -49,6 +49,8 @@ macro_rules! rpc_req {
 
 pub type JsonRpcResponseFut =
     Box<dyn Future<Item = (JsonRpcRemoteAddr, JsonRpcResponseEnum), Error = String> + Send + 'static>;
+pub type JsonRpcTransportErrFut =
+    Box<dyn Future<Item = (JsonRpcRemoteAddr, JsonRpcResponseEnum), Error = JsonRpcTransportError> + Send + 'static>;
```

Comment:
As discussed please remove `JsonRpcTransportErrFut` and add ```
pub type JsonRpcResponseFut =
    Box<dyn Future<Item = (JsonRpcRemoteAddr, JsonRpcResponseEnum), Error = String> + Send + 'static>;
``` again but make `Error = JsonRpcError`

Reply:
done